stream/src/main/java/com/annimon/stream/Stream.java

```diff
@@ -31,30 +31,6 @@
     }
 
     /**
-     * Creates a {@code Stream} from {@code List}.
-     *
-     * @param <T> the type of the stream elements
-     * @param list  the list with elements to be passed to stream
-     * @return the new stream
-     */
-    public static <T> Stream<T> of(final List<? extends T> list) {
```

Comment:
I found removing public method breaks binary-level compatibility (even if calls to this method compiles successfully).
https://docs.oracle.com/javase/specs/jls/se7/html/jls-13.html#jls-13.4.12

Should add delegation method to keep compatibility:

```java
// remove this method on next version contains incompatible change.
public static <T> Stream<T> of(final List<? extends T> list) {
    return of((Iterable) list);
}
```